src/garage/torch/algos/_utils.py

```diff
@@ -53,26 +53,26 @@ def make_optimizer(optimizer_type, module, **kwargs):
     return optimizer_type(module.parameters(), **opt_args)
 
 
-def compute_advantages(discount, gae_lambda, max_path_length, baselines,
+def compute_advantages(discount, gae_lambda, max_path_length, value_funcs,
                        rewards):
     """Calculate advantages.
 
     Advantages are a discounted cumulative sum.
 
-    Calculate advantages using a baseline (value function) according to
-    Generalized Advantage Estimation (GAE)
+    Calculate advantages using value functions according to Generalized
+        Advantage Estimation (GAE)
 
     The discounted cumulative sum can be computed using conv2d with filter.
     filter:
         [1, (discount * gae_lambda), (discount * gae_lambda) ^ 2, ...]
         where the length is same with max_path_length.
 
-    baselines and rewards are also has same shape.
-        baselines:
-        [ [b_11, b_12, b_13, ... b_1n],
-          [b_21, b_22, b_23, ... b_2n],
+    value functions and rewards are also has same shape.
```

Comment:
these values are still called a baseline. 

see https://arxiv.org/abs/1506.02438 for a reference of where this terminology comes from.